webpack.config.js

```diff
@@ -38,27 +51,28 @@ module.exports = {
     extensions: ['', '.js', '.jsx']
   },
   plugins: [
-    new ExtractTextPlugin('./dist/app.css'),
+    new ExtractTextPlugin('static/app.css'),
     new CopyWebpackPlugin([
-        { from: 'images', to: 'dist' }
-    ])
+        { from: 'images', to: 'static' }
+    ]),
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV || 'development')
+    })
   ]
 };
 
 
 if (process.env.NODE_ENV === 'production') {
-  module.exports.devtool = '#source-map'
+  module.exports.devtool = '#source-map';
   module.exports.plugins = (module.exports.plugins || []).concat([
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: '"production"'
-      }
-    }),
     new webpack.optimize.UglifyJsPlugin({
       compress: {
         warnings: false
+      },
+      output: {
+        semicolons: false
```

Comment:
Newlines cost as much as semicolons in terms of filesize, but with newlines, stacktraces from production builds are a little more useful since not everything is put into a few lines.